site.config.js

```diff
@@ -22,6 +22,9 @@ const siteConfig = {
     title: 'Terra Dev Site',
     locales: ['en'],
     bidirectional: false,
+    extensions: {
```

Comment:
Right now, the default site config is the best documentation we have on the site options. We should add the `extensions` key and a quick blrub on its usage and assign a default.
https://github.com/cerner/terra-dev-site/blob/master/src/config/site.config.js#L4

Reply:
I added extensions to the default site config with a blurb about usage. I have it commented out because there's no 'right' github url default.